src/frontend/src/lib/components/ui/RoundedIcon.svelte

```diff
@@ -5,7 +5,8 @@
 	export let iconStyleClass = '';
 </script>
 
-<div class="relative">
-	<div class="aspect-square w-12 rounded-full bg-brand-subtle-alt" />
-	<svelte:component this={icon} styleClass={`inset-center ${iconStyleClass}`} />
+<div
+	class="relative flex items-center justify-center rounded-full bg-white p-3 ring-2 ring-brand-subtle"
```

Comment:
- instead of `bg-white`, now we can use `bg-primary`

- i was under the impression that the ring was color `brand-subtle-alt` (looking at a specific component I am modifying now). Just to double-confirm please

Reply:
> bg-primary

Done.

> brand-subtle-alt

Are you sure? Looks incorect to me.

<img width="1536" alt="Capture d’écran 2024-11-13 à 14 20 09" src="https://github.com/user-attachments/assets/d1806e38-0bc5-4ccf-beaf-b3006cd9e043">

Reply:
ah wow! then no! boh! I thought it was `#D1E3FF` , that we mapped as `bg-brand-subtle-alt`, but i can be mistaken, no worries, all good, thank for checking